src/pages/signup-affiliates/components/_account-details.tsx

```diff
@@ -101,8 +164,56 @@ const AccountDetails = () => {
             label: localize('Postal/Zip code'),
             placeholder: 'Postal/Zip code',
             required: true,
+            error: postcode_error_msg,
+            value: postal_code,
         },
     ]
+    const handleValidation = (e) => {
```

Comment:
Can we rename is as `handleInput` since this is doing value setting and validation at thesame time.